python/doc/examples/data_analysis/quick_start_point_and_sample.ipynb

```diff
@@ -0,0 +1,992 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[5, 5, 5]"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "p4 = [5] * 3\n",
+    "p4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also create a `Sample` from a list of `Point`s."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>v0</TH><TH>v1</TH></TR>\n",
+       "<TR><TD>0</TD><TD>2.0</TD><TD>3.0</TD></TR>\n",
+       "<TR><TD>1</TD><TD>0.0</TD><TD>1.0</TD></TR>\n",
+       "<TR><TD>2</TD><TD>4.0</TD><TD>9.0</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=3 dimension=2 data=[[2,3],[0,1],[4,9]]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sample = ot.Sample ([p1 , p2 , p3 ])\n",
+    "sample"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can loop over the points in a sample, using a list comprehension. In the following example, we compute the euclidian norm of the points in the previous sample. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[3.605551275463989, 1.0, 9.848857801796104]"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "[point.norm() for point in sample]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also create a `Sample` based on a `Point`, repeated three times."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>v0</TH><TH>v1</TH><TH>v2</TH></TR>\n",
+       "<TR><TD>0</TD><TD>5.0</TD><TD>5.0</TD><TD>5.0</TD></TR>\n",
+       "<TR><TD>1</TD><TD>5.0</TD><TD>5.0</TD><TD>5.0</TD></TR>\n",
+       "<TR><TD>2</TD><TD>5.0</TD><TD>5.0</TD><TD>5.0</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=3 dimension=3 data=[[5,5,5],[5,5,5],[5,5,5]]"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sample = ot.Sample ([p4] * 3)\n",
+    "sample"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Perhaps the simplest way to create a non trivial `Sample` is using a nested list of floats."
```

Comment:
```suggestion
    "Nested lists of floats may be the easiest way to create non-trivial Samples."
```

Reply:
I do not like the idea of `Samples` because this class does not exist. And writing `Sample`s is not so clean. So I used:

"A nested list of floats is the easiest way to create a non-trivial `Sample`."

Reply:
> Third part of my review.

Balzac was not so verbose, after all...